docs/README.md

```diff
@@ -132,11 +144,13 @@ $ ip route
 You should now be able to reach that node over the wireguard tunnel.
 
 ```sh
-$ ping -c 1 10.200.0.2
+$ ping 10.200.0.2
 PING 10.200.0.2 (10.200.0.2) 56(84) bytes of data.
 64 bytes from 10.200.0.2: icmp_seq=1 ttl=64 time=7.63 ms
 ```
 
+You can also create new zones, as well as viewing and deleting peers in the web UI by visiting the URL of the host you added in your `/etc/hosts` file. For example, `https://apex.local/`.
```

Comment:
It's not really clear how to make use of multiple zones right now. I wouldn't mention it.